frontend/src/components/MyReviewCard.js

```diff
@@ -2,11 +2,35 @@ import styled from "styled-components/macro";
 import { FaStar } from "react-icons/fa";
 import { MdDeleteForever, MdModeEdit } from "react-icons/md";
 import StyledIconButton from "./StyledIconButton";
+import { useMutation, useQueryClient } from "react-query";
+import { removeReview } from "../service/ReviewService";
+import AuthContext from "../context/AuthContext";
+import { useContext } from "react";
+import { useHistory } from "react-router-dom";
 
 export default function MyReviewCard({ review }) {
+  const queryClient = useQueryClient();
+  const history = useHistory();
+  const { token } = useContext(AuthContext);
+
+  const deleteReview = useMutation(
+    () => {
+      return removeReview(token, review.reviewId);
+    },
```

Comment:
Imho implicit return is more readable.
```suggestion
    () => removeReview(token, review.reviewId),
```